tasks/get_peadm_config.rb

```diff
@@ -11,9 +11,9 @@ class GetPEAdmConfig
   def initialize(params); end
 
   def execute!
-    # if there is no 'PE HA Replica' node group, it's not a peadm-configured cluster.
-    replica_group = groups.data.find { |obj| obj['name'] == 'PE HA Replica' }
-    if replica_group
+    # if there is no 'PE Primary A' node group, it's not a peadm-configured cluster.
+    peadm_primary_a_group = groups.data.find { |obj| obj['name'] == 'PE Primary A' }
+    if peadm_primary_a_group
```

Comment:
I believe this is intentional. I think we should keep it as is for now, and the rest looks fine.

Reply:
Do you know why? The thing is that 'PE HA Replica' is a node group present in all installations of PE that have a replica, regardless of whether peadm was used to install the installation or not. So if the test is intended to verify whether this installation was generated using peadm, checking for that won't work. It would also exclude any peadm installation that did not have a replica.

Reply:
@jpartlow The test is about the group "PE Primary A", not "PE HA Replica". "PE Primary A" only exists in PEADM-managed clusters. Or did I misunderstand your question?

Reply:
@timidri er, right, that's why I changed it to be 'PE Primary A' in this patch :). I was double checking that there wasn't some reason it had previously been set to test 'PE HA Replica'.

Reply:
Oh oops, I should have had a better look. You are right, seems the previous version was a bug, not sure what I was thinking when putting that code there. Thanks for the catch!

Reply:
:+1: The node groups are confusing; that's why I was double checking :)